onnxruntime/core/common/cpuid_info.cc

```diff
@@ -64,56 +63,54 @@ static inline int XGETBV() {
 CPUIDInfo CPUIDInfo::instance_;
 
 
-common::Status CPUIDInfo::Init() {
-
+CPUIDInfo::CPUIDInfo() {
 #ifdef CPUINFO_INCLUDED
-  if (!cpuinfo_initialize()) {
-    // Unfortunately we can not capture cpuinfo log!!
-    return ORT_MAKE_STATUS(SYSTEM, FAIL, "Failed to initialize cpuinfo");
-  }
+    if (!cpuinfo_initialize()) {
+      // Unfortunately we can not capture cpuinfo log!!
+      ORT_THROW("Failed to initialize CPU info.S");
+    }
```

Comment:
nit: is the trailing 'S' intentional?

Reply:
that was evidence of my stupidity and now that evidence has been destroyed.